manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -22,3 +23,25 @@ spec:
           severity: warning
         annotations:
           message: "CoreDNS Health Checks are slowing down (instance {{ $labels.instance }})"
+      - alert: CoreDNSErrorsHigh
+        expr: |
+          (sum(rate(coredns_dns_response_rcode_count_total{rcode="SERVFAIL"}[5m]))
+            /
+          sum(rate(coredns_dns_response_rcode_count_total[5m])))
+          > 0.03
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          message: "CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of requests."
+      - alert: CoreDNSErrorsHigh
```

Comment:
I think having only one `CoreDNSErrorsHigh` alert would be best, in order to remove duplicate alerts and thus minimize "alert fatigue". As far as which value to use (.03 vs .01), I don't have any strong opinions on which one to use. Thoughts?

Reply:
I prefer to keep the `0.01` one since I don't think it is really necessary to add a critical level alert here.

Reply:
sounds good to me.

Reply:
@RiRa12621  Hello, would you mind taking a look at this? Which alert should we use? The critical one or the warning one?

Reply:
+1 for warning from me
/cc @jewzaam